etc/avocado/conf.d/vt_joblock.conf

```diff
@@ -0,0 +1,6 @@
+[plugins.vtjoblock]
+# Directory where the lock file will be located. Avocado should have permission
+# to write to this directory.
+dir=/tmp
+# Whether to show the acquired job lock file location on the Avocado UI
+show_lock_location=False
```

Comment:
I don't think this needs to be configurable and I'd make True as default. I think it's a useful message related to avocado-vt executions.